Tests/SecureXPCTests/Client & Server/Round-trip Integration Test.swift

```diff
@@ -20,7 +20,7 @@ class RoundTripIntegrationTest: XCTestCase {
         anonymousServer.start()
     }
 
-    func testSendWithMessageWithReply() throws {
+    func testSendWithMessageWithReply_Sync() throws {
```

Comment:
I think it's reasonable to just completely replace these closure-based tests.

In my own work, I've pretty much completely abandoned the XCTest continuation APIs in favour of async/await, even for old projects that don't use it in the production code (I just add wrappers in my test bundles, which makes test writing easier and more correct).

Reply:
I'm going to leave them for now as the behavior differs for two of them depending on whether they have a closure provided or not, but point taken.